skype/skype-ps/skype/Set-CsOnlineVoicemailUserSettings.md

```diff
@@ -82,14 +85,102 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -VoicemailEnabled
-The VoicemailEnabled parameter represents whether to enable voicemail service.
+### -CallAnswerRule
+The CallAnswerRule parameter represents the value of the call answer rule, which can be any of the following:
+
+- DeclineCall
+- PromptOnly
+- PromptOnlyWithTransfer
+- RegularVoicemail
+- VoicemailWithTransferOption
+
+```yaml
+Type: Object
+Parameter Sets: (All)
+Aliases: 
+Applicable: Microsoft Teams, Skype for Business Online
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DefaultGreetingPromptOverwrite
+The DefaultGreetingPromptOverwrite parameter represents the contents that overwrite the default normal greeting prompt.
+If user normal custom greeting is not set and DefaultGreetingPromptOverwrite is not empty, voicemail service will play this overwrite greeting instead of default normal greeting in voicemail deposit scenario.
```

Comment:
```suggestion
If the user's normal custom greeting is not set and DefaultGreetingPromptOverwrite is not empty, the voicemail service will play this overwrite greeting instead of the default normal greeting in the voicemail deposit scenario.
```